src/map.cpp

```diff
@@ -964,12 +967,10 @@ vehicle *map::displace_vehicle( tripoint &p, const tripoint &dp )
         }
 
         if( psg->pos() != part_pos ) {
-            debugmsg( "Passenger/part position mismatch: passenger %d,%d,%d, part %d %d,%d,%d",
-                      g->u.posx(), g->u.posy(), g->u.posz(),
-                      prt,
-                      part_pos.x, part_pos.y, part_pos.z );
-            veh->parts[prt].remove_flag( vehicle_part::passenger_flag );
-            continue;
+            add_msg( m_debug, "Passenger/part position mismatch: passenger %d,%d,%d, part %d %d,%d,%d",
```

Comment:
Why did you change this? Messages with `m_debug` are *only* shown when debug mode is enabled, which must be done manually And messages like this can easily be missed. `debugmsg` messages are always shown and can't be missed. If this message indicates a serious problem, it should announce this loudly.

Reply:
I don't think it's a severe problem at this point. It can happen when a vehicle's skew changes due to parts splitting off. I did my best to correct for it, but it still happens sometimes for reasons I don't understand.